src/dstore/pmix_esh.c

```diff
@@ -591,18 +591,19 @@ int _esh_init(pmix_info_t info[], size_t ninfo)
         }
 
         if (NULL ==_base_path) {
```

Comment:
I don't like that `_base_path` is used with 2 different meanings here - this may lead to a confusion.
Lets use the other temp variable `_dstor_tmpdir` for info-key processing, and `_base_path` will only represent the final path that this server will use.